### DISCUSSION TITLE
42

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#796] [Comments]
http://jojoldu.tistory.com/42

**Comment by jojoldu** [comment]
작성자: seungdols

정말 좋은 플러그인 소개 감사합니다. ㅎ 저도 설치해서 써봐야겠네요.

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

seungdols님 매번 방문해주셔서 감사합니다^^
좋은 플러그인인것은 분명해서 꼭 써보시길 추천드립니다

**Comment by jojoldu** [comment]
작성자: dotorify

이런 좋은 플러그인이 있었다니요! 당장 설치해봐야겠네요. 감사합니다

**Comment by djpibo** [comment]
keySet()은 key만 반환하지만
entrySet()은 key, value 모두 반환해주기 때문에 

저 로직에선 이왕이면 다옹치마로 entrySet()을 사용해서 가져와야 한다는 거군요.
코드 효율성도 체크해주는 정적 분석 솔루션,,, 장난 아니네요
